packages/aws-cdk/test/integ/cli/bootstrapping.integtest.ts

```diff
@@ -1,52 +1,175 @@
-import { cloudFormation, deleteStacks } from './aws-helpers';
-import { cdk, cleanupOldStacks, fullStackName, prepareAppFixture } from './cdk-helpers';
+import { cloudFormation, deleteStacks, s3 } from './aws-helpers';
+import { cdk, cdkDeploy, cleanupOldStacks, deleteableStacks, fullStackName, prepareAppFixture } from './cdk-helpers';
 
-jest.setTimeout(600 * 1000);
+jest.setTimeout(600_000);
+
+// Needs to not start with the ACTUAL fullstack name otherwise cleanupOldStacks will pick it up.
+const BOOTSTRAP_STACK_PREFIX = `toolkit-${fullStackName('stack')}`;
+const BUCKETS_TO_DELETE = new Array<string>();
+const QUALIFIER = randomString();
 
 beforeAll(async () => {
   await prepareAppFixture();
 });
 
 beforeEach(async () => {
   await cleanupOldStacks();
+  BUCKETS_TO_DELETE.splice(0, BUCKETS_TO_DELETE.length);
 });
 
 afterEach(async () => {
   await cleanupOldStacks();
+  await cleanupBootstrapStacks();
```

Comment:
It's hard to describe how confusing it is to read that `cleanupOldStacks()` somehow doesn't cleanup up the bootstrap stacks. Can you just use the same prefix for the bootstrap stacks that you do for the "old" stacks, so you don't have to do these separate calls? Or at least rename `cleanupOldStacks` to something like `cleanupApplicationStacks()`, or something?

Reply:
Mmmyeah. The bootstrap stacks need extra work (in cleaning the s3 buckets) that the existing method didn't do. You're right, I can just make the other cleanup function smarter.